Backend/Core/Services/CategoryService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AutoMapper;
+using Core.DTO.Category;
+using Core.Entities;
+using Core.Exceptions;
+using Core.Helpers;
+using Core.Interfaces;
+using Core.Interfaces.CustomService;
+using Core.Resources;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Core.Services
+{
+    internal class CategoryService : ICategoryService
+    {
+        private readonly IRepository<Category> _categoryRepository;
+        private readonly IMapper _mapper;
+
+        public CategoryService(
+            IRepository<Category> categoryRepository, 
+            IMapper mapper)
+        {
+            _categoryRepository = categoryRepository;
+            _mapper = mapper;
+        }
+
+        public async Task CreateCategoryAsync(CreateCategoryDTO createTripDTO)
+        {
+            if (createTripDTO != null)
```

Comment:
Null cannot come here with the current logic. If we use this method somewhere else, we will add this check